orangecontrib/text/widgets/owimporttexts.py

```diff
@@ -0,0 +1,642 @@
+"""
+Import Texts
+-------------
+
+Import texts 'into' canvas session from a local file system
+"""
+import sys
+import os
+import enum
+import warnings
+import logging
+import traceback
+
+from types import SimpleNamespace as namespace
+
+import numpy as np
+
+from AnyQt.QtCore import Qt, QEvent, QFileInfo, QThread
+from AnyQt.QtCore import pyqtSlot as Slot
+from AnyQt.QtGui import QStandardItem, QDropEvent
+
+from AnyQt.QtWidgets import (
+    QAction, QPushButton, QComboBox, QApplication, QStyle, QFileDialog,
+    QFileIconProvider, QStackedWidget, QProgressBar, QWidget, QHBoxLayout,
+    QVBoxLayout, QLabel
+)
+
+import Orange.data
+
+from Orange.widgets import widget, gui, settings
+from Orange.widgets.utils.filedialogs import RecentPath
+from Orange.widgets.utils.concurrent import (
+    ThreadExecutor, FutureWatcher, methodinvoke
+)
+from Orange.widgets.widget import Output
+
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.widgets.utils.readers import TextScan
```

Comment:
All right, I am moving there. But are you sure that `text_import.py` is a proper file name? Maybe a name without an underscore?